src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -677,7 +677,7 @@ public void setRequestInterceptor(RequestInterceptor interceptor) {
      * @param resource the resource for which the new token has to be obtained
```

Comment:
our linter is pretty strict with docstrings so with the overloaded method if you can put a @param sharedLink <sharedLink description> it will make then make the linter pass